Kickstarter-iOS/Views/Cells/PledgeAmountCell.swift

```diff
@@ -73,39 +77,6 @@ final class PledgeAmountCell: UITableViewCell, ValueCell {
 
 // MARK: - Styles
 
-private func inputStackViewStyle(_ isAccessibilityCategory: Bool) -> ((UIStackView) -> UIStackView) {
-  return { (stackView: UIStackView) in
-    let alignment: UIStackView.Alignment = (isAccessibilityCategory ? .leading : .center)
-    let axis: NSLayoutConstraint.Axis = (isAccessibilityCategory ? .vertical : .horizontal)
-    let distribution: UIStackView.Distribution = (isAccessibilityCategory ? .equalSpacing : .fill)
-    let spacing: CGFloat = (isAccessibilityCategory ? Styles.grid(1) : 0)
-
-    return stackView
-      |> \.alignment .~ alignment
-      |> \.axis .~ axis
-      |> \.distribution .~ distribution
-      |> \.spacing .~ spacing
-  }
-}
-
-private let labelStyle: LabelStyle = { (label: UILabel) in
-  label
-    |> \.adjustsFontForContentSizeCategory .~ true
-    |> \.font .~ UIFont.ksr_headline()
-    |> \.numberOfLines .~ 0
-    |> \.text %~ { _ in Strings.Your_pledge_amount() }
-}
-
-private let rootStackViewStyle: StackViewStyle = { (stackView: UIStackView) in
-  stackView
-    |> \.axis .~ NSLayoutConstraint.Axis.vertical
-    |> \.isLayoutMarginsRelativeArrangement .~ true
-    |> \.layoutMargins .~ UIEdgeInsets(
-      top: Styles.grid(2), left: Styles.grid(4), bottom: Styles.grid(3), right: Styles.grid(4)
```

Comment:
If these margins are standard for all of the cells in the pledge VC, I wonder if it might make sense to put them on the cell's `contentView`s shared checkout style instead, so other cells can also have the same layout margins?

Reply:
Do you mean simply moving `layoutMargins` to its own style (`checkoutCellContentViewStyle`) and removing it from here?

I'm not sure if I'm doing it wrong but it doesn't work for me :/

Reply:
Hmm I wonder if you need to set `preservesSuperviewLayoutMargins` on the stack view? 🤔or alternatively you might need to constraint the stack view to the `contentView`s margins. This isn't super important but I just wanted to see if there was a way to share layout margins among all the pledge cells.